Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -145,50 +170,73 @@ private void SetControls()
 
         public void RefreshALMSolutionSettings()
         {
-            if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.QC && QCRadioButton.IsChecked == false)
+            if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC && QCRadioButton.IsChecked == false)
             {
                 QCRadioButton.IsChecked = true;
                 PasswordTextBox.Password = "";
                 UserNameTextBox.Text = "";
                 StyleRadioButtons();
             }
-            else if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.RQM && RQMRadioButton.IsChecked == false)
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.RQM && RQMRadioButton.IsChecked == false)
             {
                 RQMRadioButton.IsChecked = true;
                 PasswordTextBox.Password = "";
                 UserNameTextBox.Text = "";
                 StyleRadioButtons();
             }
-            else if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.RALLY && RallyRadioButton.IsChecked == false)
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.Jira && JiraRadioButton.IsChecked == false)
+            {
+                JiraRadioButton.IsChecked = true;
+                PasswordTextBox.Password = "";
+                UserNameTextBox.Text = "";
+                StyleRadioButtons();
+            }
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.RALLY && RallyRadioButton.IsChecked == false)
             {
                 RallyRadioButton.IsChecked = true;
                 PasswordTextBox.Password = "";
                 UserNameTextBox.Text = "";
                 StyleRadioButtons();
             }
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.Qtest && qTestRadioButton.IsChecked == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)